build.gradle

```diff
@@ -27,12 +27,11 @@ repositories {
     maven { url 'http://repo.spring.io/release'}
 }
 
-
 dependencies {
 
-    ext.dropwizardVersion = '0.8.1'
+    ext.dropwizardVersion = '0.8.4'
     ext.springIoPlatformVersion = '1.1.2.RELEASE'
-    ext.jerseyVersion = '1.19'
+    ext.jerseyVersion = '2.22.1'
```

Comment:
I would simply remove this - the jersey version should come from the DW dependency.